### PR TITLE
VFB-233: deleted parcels should be hidden by default

### PR DESCRIPTION
**Pull request by mtharkins**
## What's changed
On the parcels page, the "Last status" filter now doesn't check "Deleted parcels" by detault

When a client is deleted, any parcels associated with them are updated to be deleted 

## Checklist
- [x] The ticket is up-to-date - Please document any deviations from the original approach if there is any.
- [x] I have documented the testing steps for QA
- [x] I have self-reviewed this PR
- [x] Make sure you've verified it works via `npm run dev`
- [x] Make sure you've verified it works via `npm run build` and `npm run start`
- [x] Make sure you've fixed all linting problems with `npm run lint_fix`
- [x] Make sure you've tested via `npm run test:e2e`

If you have made any changes to the database...
  - [ ] The migration files are up-to-date with my final set up (`npx supabase db diff -f <name_of_migration>` should create nothing at this point)
  - [ ] I have updated the typescript definitions for the database with `npm run db:local:generate_types`
  - [ ] I have modified the seed in `supabase/seed/seed.mts` if appropriate
  - [ ] If I have modified the seed, I have also generated the seed with `npm run db:generate_seed` 
  - [ ] With my final set up, I can run `npm run dev:reset_supabase` without any errors.
  - [ ] (Just before merging the ticket) I have updated the timestamps of the new migration files so that they are after all existing migration files.
  - [ ] If I have added constraints e.g. `set not null`, I have taken reasonable measures to ensure constraint violations do not happen when the migration occurs. 
  - [ ] I have checked that migration can happen successfully without resetting the database, doing the following.
    - Make sure you have rebased your branch onto dev or merged dev into your branch.
    - Checkout the dev branch
    - Run `npm run post_checkout` to reset the database, including the seed data.
    - Checkout your branch
    - Run `npx supabase migrations up --include-all --local` to run all outstanding migration files
    - Check that the resulting database is what you expect, bar any seed data changes.

### CODE REVIEW

**Comment by 8kristy** [comment]
Looks good :) Just need to run the linter and fix the e2e tests

File: src/app/clients/clientsTable/format.ts [hunk 1]
```diff
@@ -1,16 +1,18 @@
...
+        case "failedClientIsActiveFetch":
+            return `Failed to determine whether client is active. Please reload. Log ID: ${error.logId}`;
+    }
+};
+
+export const getDeleteClientErrorMessage = (error: DeleteClientError): string => {
+    switch (error.type) {
+        case "failedClientDeletion":
+            return `Failed to delete client. Please try again later. Log ID: ${error.logId}`;
+        case "eventInsertionFailed":
```
Comment (8kristy) [inline src/app/clients/clientsTable/format.ts]: Think you need to update DeleteClientErrorType to include this new error

Comment (mtharkins) [reply src/app/clients/clientsTable/format.ts]: This commit acceidentally excluded the Type update as the file wasn't saved locally, it should be included now.

**Comment by mtharkins** [comment]
I think the line endings issue has now been resolved as the lint & E2E tests are passing.

File: src/app/clients/deleteClient.ts [hunk 2]
```diff
@@ -14,6 +15,8 @@ const deleteClient = async (clientId: string): Promise<DeleteClientResponse> =>
         clientid: clientId,
     });
 
+    getClientParcelsDetails(clientId);
```
Comment (stonelink) [inline src/app/clients/deleteClient.ts]: Hi @mtharkins I've just started looking at the outstanding PRs.

I _think_ that this isn't needed - you've made the necessary change in ClientsPage.tsx and I don't think this line does anything. I'm guessing that this was an earlier attempt that you changed your mind about.

Please let me know if I've misunderstood, but for now I'll revert this file.

**Review by stonelink — APPROVED** [review]

**Pull request merged**